src/controllers/funds.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
             }
             const filteredItem = await Funds.find(filter);
             if (filteredItem.length === 0) {
-                return res.status(500).json({
+                return res.status(404).json({
```

Comment:
Better to delete this condition altogether and return an empty array showing that no funds met the specified conditions.